### PR TITLE
Upgrade to react-admin v5

### PR DESCRIPTION
**Pull request by djhi**
Tests won't pass until we provide a token in the repository actions settings

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: "Test - action"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  typecheck:
```
Comment (fzaninotto) [inline .github/workflows/test.yml]: I'd make only one job with both typecheck and test, as most of the job time is spent initializing

**Comment by fzaninotto** [comment]
I have a few UI issues.

- The dark theme isn't legible in some pages:
 
image

- we need to add the theme switcher back to the appbar, and default to light theme
- The user menu isn't vertically centered in the appbar
image
- the tanstack query devtools icon is too large

**Comment by fzaninotto** [comment]
2 things are still missing:

- default to light theme
- the tanstack query devtools icon is too large

**Comment by djhi** [comment]
> default to light theme

Why not let the user system decide?

**Comment by fzaninotto** [comment]
> Why not let the user system decide?

Because the dark theme of this app isn't very nice IMHO

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**